src/main/java/net/huskycraft/blockyarena/BlockyArena.java

```diff
@@ -109,6 +134,7 @@ private void createManagers() {
     private void createDirectories() {
         arenaDir = Paths.get(getConfigDir().toString() + "/arenas");
         kitDir = Paths.get(getConfigDir().toString() + "/kits");
+        defaultConfig = Paths.get(getConfigDir().toString(), "default.json");
```

Comment:
To be consistent with Sponge's style, we're using .conf for configs.

Reply:
Fixed